scripts/smoke-test-linux.sh

```
@@ -0,0 +1,305 @@
+"$SHED_BIN" create "$SHED_NAME" --local-dir "$LOCAL_DIR" --image "$SMOKE_IMAGE" || fail "shed create"
+CREATED=true
+endsection
+
+section "Step 7: shed exec — confirm /workspace mount"
+EXPECTED="smoke marker"
+OUTPUT=""
+# Retry briefly: `shed create` returns once the VM is Running but
+# sshd inside the guest needs a moment more to bind :22. Five
+# attempts at 2 s spacing has been enough on every environment
+# tested so far; bump if a slower host needs it.
+for attempt in 1 2 3 4 5; do
```

Shellcheck (0.11.0)

[warning] attempt appears unused. Verify use (or export if used externally).

(SC2034)